py/bodynoline.py

```diff
@@ -15,6 +16,47 @@ def _bodynoline(title, entries):
         The HTML consists of tables to orient each entry within the section.
 
     Raises:
-        None
+        TypeError - If the object type of elements within the entry aren't acceptable types.
```

Comment:
write the checks: within body, needs to be content or image, within content, needs to be text, link, or linebreak.

also add check for title = string

Reply:
added specific TypeError scenarios